### ISSUE TITLE
Length of input text doesn't match what's shown in the samples

### ISSUE THREAD

**Issue by uberdog** [DevProgress/my-vote-matters#49]
Typing the text that's shown in a sample shows that the actual limit is much shorter.
<img width="1382" alt="screen shot 2016-10-24 at 10 37 27 pm" src="https://cloud.githubusercontent.com/assets/1243310/19671133/9cec583e-9a3a-11e6-93c4-44f3556d93c4.png">

**Comment by durka** [comment]
This is because the caption as you enter it is limited to 140 characters, taking into account the automatically added suffix " (share yours at myvotematters.today)". One way to fix this would be to shorten that suffix (rephrase and/or buy a shorter domain name).

**Comment by durka** [comment]
Another option would be to relax the limit, but disable sharing to Twitter if your caption is too long.

**Comment by elevati0n** [comment]
Hi, I'm new here but I'm eager to help.  Sharing to twitter seems important, but so does having a message.  In order to keep both:
- Could we possibly have the text included as part of the photo when it goes over 140 characters?  

If the text box is required, I wouldn't disallow twitter posting, I'd rather enforce the character limit 
 using javascript OR CSS i believe
- The input box could turn red and go into disable state when more than 140(-(share yours at myvotematters.today).length()) characters are entered.  

I can try to code the second option this weekend.

**Comment by durka** [comment]
Yeah, there is technically no limit to the text length as far as rendering it on the image is concerned, though it will get too small to read pretty fast. (The [line spacing calculation](https://github.com/DevProgress/my-vote-matters/blob/809d734e33e0ad87a5fe8001f1fc81fad46489d6/js/main.js#L478) could use some work too.)

I like the option of putting all the text in the image but abbreviating the twitter message (just put in an ellipsis I guess). Also, #58 will add some more breathing room.

**Comment by elevati0n** [comment]
OR we could truncate the comments that people do have on twitter at the character limit and elipisseis and then append a link to our site if ppl want to see the matching FULL text and image.

**Comment by elevati0n** [comment]
assuming the placards actually exist on our site and they have RESTful URIs...

**Comment by durka** [comment]
> OR we could truncate the comments that people do have on twitter at the character limit and elipisseis

Yeah that's what I was thinking, and then you render the entire caption on the image.

> assuming the placards actually exist on our site and they have RESTful URIs...

Haha, this is actually a frontend-only site. The images don't exist anywhere outside of the canvas on the page.

**Comment by elevati0n** [comment]
Yeah I figured the limited scope out after I wrote my comment and realized I was being way too ambitious haha.
I'm going to play around with a couple ideas and some CSS tweaks and such while I get acquanited with the project.  I was thinking that instead of having the textbox form switch to disable after ~140 characters instead have it switch to a different form which effectively uploads the 'file download' version of the whole image + text and makes the card neatly fit into a ready to share image (and still put a URI link in the text of the body).
